next/components/ui/PartnerCard/PartnerCard.tsx

```diff
@@ -42,6 +42,7 @@ export const Partner = ({ className, title, pageLink, logo, alt, featured }: Par
         className={cx(className, 'flex items-center space-x-[9px]', {
           'mt-[6px] lg:mt-[13px]': featured,
         })}
+        stretched
         href={pageLink?.url ?? '#'}
       >
         <Globe />
```

Comment:
Please replace it by icon from `assets/icons`. If Globe is missing, add it there (and change fills and paths to `currentColor`)